ui/components/connections/metadata.js

```diff
@@ -199,6 +199,13 @@ const KubernetesMetadataFormatter = ({ meshsyncControllerState, connection, meta
                     secondary={<Link title={metadata.server}>{metadata.server}</Link>}
                   />
                 </ListItem>
+                <ListItem>
+                  <ListItemText
+                    primary=" Connection Id"
+                    secondary={<FormatId id={connection.id} />}
+                    style={{ marginTop: '-12px' }}
```

Comment:
why this styling?

Reply:
For some reason the alignment of this item w.r.t other items was not right when I was making the changes I'll have a look and test the change again.

Reply:
@Divyansh200102 did you look? use align center or any proper css because this will create issues since if anything around it changes again the style will be affected due to negative px

Reply:
@sudhanshutech i tried removing the styling entirely and this is how it's coming out.
![image](https://github.com/user-attachments/assets/58ba53b8-76b7-47ff-b4ef-6dcc09073c62)

I don't know if this is alignment issue is happening on my end but the "Updated at" and "Connection Id " are coming out at this if possible can you check this change on your end?